pint_server/pint_db_migrate/README

```diff
@@ -0,0 +1,4 @@
+This is a database migration repository.
+
+More information at
+http://code.google.com/p/sqlalchemy-migrate/
```

Comment:
Probably should also add some details about how this is expected to be used for the pint DB project specific, e.g. a high level overview of the workflow for those not familiar with it ;-)

Reply:
this file is generated by the framework. Let up update the README.rst at the top with the appropriate instructions instead.